app/config/locale/templates/email-webhook-failed.tpl

```diff
@@ -0,0 +1,20 @@
+<p>Hi <strong>{{user}}</strong>,</p>
+<p>Your webhook <strong>{{webhook}}</strong> on project <strong>{{project}}</strong> has been paused after {{attempts}} consecutive failures.</p>
+<p>Webhook Endpoint: <strong>{{url}}</strong></p>
+<p>Error: <strong>{{error}}</strong></p>
+<p>To restore your webhook's functionality and reset attempts, we suggest to follow the below steps:</p>
+<ol>
+    <li>Examine the logs of both Appwrite Console and your webhook server to identify the issue.</li>
+    <li>Investigate potential network issues and use webhook testing tools to verify expected behaviour.</li>
+    <li>Ensure the webhook endpoint is reachable and configured to accept incoming POST requests.</li>
+    <li>Confirm that the webhook doesn't return error status codes such as 400 or 500.</li>
+</ol>
+<p>After the issue is resolved, please make sure to re-enable the webhook directly through the webhook settings.</p>
+
+<table border="0" cellspacing="0" cellpadding="0" style="padding-top: 10px; padding-bottom: 10px; margin-top: 32px">
```

Comment:
Maybe not in this scope, but we really need to find a way to reuse this button style. It makes really bad maintenance design to have it like this, we will not be able to have any two button that look the same if we keep it like this.

Reply:
Created a separate issue to resolve this.